### ISSUE TITLE
Prevent flooding the log file with `ERROR : Hit layaway file limit. Not writing to layaway file`

### ISSUE THREAD

**Issue by itsderek23** [scoutapp/scout_apm_ruby#144]
If an error occurs writing to the log file, each Ruby process will log an error continually to a file:

```
ERROR : Hit layaway file limit. Not writing to layaway file
```

This can eventually generate a multi-GB log file for apps w/many Ruby processes.

Instead, we should likely only log this once.

**Comment by dlanderson** [comment]
Closed via  c342465

**Comment by cschneid** [comment]
This will be included in 2.3.5
